src/Migrations/Version20180626065141.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20180626065141 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
```

Comment:
What happened with these spaces? Same for all `DoctrineMigrations`.
```suggestion
    public function up(Schema $schema): void
```